app/components/Views/confirmations/SendFlow/AddressList/AddressList.test.tsx

```diff
@@ -10,15 +10,25 @@ const MOCK_ACCOUNTS_CONTROLLER_STATE = createMockAccountsControllerState([
   MOCK_ADDRESS,
 ]);
 
-jest.mock('../../../../../core/Engine', () => ({
-  context: {
-    KeyringController: {
-      state: {
-        keyrings: [],
+jest.mock('../../../../../core/Engine', () => {
+  // eslint-disable-next-line @typescript-eslint/no-shadow
+  const { MOCK_ACCOUNTS_CONTROLLER_STATE } = jest.requireActual(
+    '../../../../../util/test/accountsControllerTestUtils',
+  );
```

Comment:
What do you think if we just rename this variable to remove the `eslint-disable` directive? For example,
```suggestion
  const { MOCK_ACCOUNTS_CONTROLLER_STATE: mockAccountsControllerState } = jest.requireActual(
    '../../../../../util/test/accountsControllerTestUtils',
  );
```

There are other files with this same directive that can also be updated in this PR

Reply:
Good idea